dali/pipeline/executor/executor.h

```diff
@@ -140,6 +140,51 @@ class DLL_PUBLIC Executor : public ExecutorBase, public WorkspacePolicy, public
 
   void SetupOutputQueuesForGraph();
 
+  template <typename Workspace>
+  void RunHelper(OperatorBase &op, Workspace &ws) {
+    constexpr size_t stage_id = static_cast<size_t>(workspace_to_op<Workspace>::value);
+    output_desc_[stage_id].clear();
```

Comment:
Do you need to clear it or it is up to op.Setup to do it?

Reply:
I think it's cleaner if we clear it here and then we avoid duplicating this call in every operator (potentially freeing ourselves from a bug that `clear` has not been called).